src/coverlet.core/coverlet.core.csproj

```diff
@@ -17,4 +16,24 @@
     <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" />
   </ItemGroup>
 
+  <!-- Nuget package properties https://docs.microsoft.com/en-us/nuget/reference/msbuild-targets -->
```

Comment:
At the moment we don't want to expose core lib as package, we discussed this one a lot of times, but for the moment we don't have a stable shape to expose. If we expose core as lib we want to do it in a proper stable way with a public "stable" contract.